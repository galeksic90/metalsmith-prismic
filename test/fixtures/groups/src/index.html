---
template: index.hbt
prismic:
  group-single:
    query: '[[:d = any(document.type, ["group-single"])]]'
  group-repeat:
    query: '[[:d = any(document.type, ["group-repeat"])]]'
---
