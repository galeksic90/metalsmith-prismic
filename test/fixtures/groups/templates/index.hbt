<html>
<body>
<ul>
{{#each prismic.group-single.results}}
    <li>
        {{{data.name.html}}}
        <dl>
        {{#each data.address.children.[0]}}
            <dt>{{@key}}</dt>
            <dd>{{{html}}}</dd>
        {{/each}}
        </dl>
    </li>
{{/each}}
</ul>
<ul>
{{#each prismic.group-repeat.results}}
    <li>
        <ul>
        {{#each data.gallery.children}}
            <li>
                {{{image.html}}}
                {{{caption.html}}}
            </li>
        {{/each}}
        </ul>
    </li>
{{/each}}
</ul>
</body>
</html>
