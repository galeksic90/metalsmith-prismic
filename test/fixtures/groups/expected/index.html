<html>
<body>
<ul>
    <li>
        <span>John Doe</span>
        <dl>
            <dt>street</dt>
            <dd><span>Hight Street</span></dd>
            <dt>zip</dt>
            <dd><span>12345</span></dd>
            <dt>city</dt>
            <dd><span>Anytown</span></dd>
        </dl>
    </li>
    <li>
        <span>Jane Doe</span>
        <dl>
            <dt>street</dt>
            <dd><span>Main Street</span></dd>
            <dt>zip</dt>
            <dd><span>67890</span></dd>
            <dt>city</dt>
            <dd><span>Sometown</span></dd>
        </dl>
    </li>
</ul>
<ul>
    <li>
        <ul>
            <li>
                <img src="https://prismic-io.s3.amazonaws.com/api-test/8cb00fd1a4e46f88f76c6b1ae8b0a910c6006e59_apple-pie.png" width="500" height="500" alt="Apple pie">
                <p><em>Apple</em> pie</p>
            </li>
            <li>
                <img src="https://prismic-io.s3.amazonaws.com/api-test/804e464860491df10a969e40e3a1404a4f261847_kiwi-pie.png" width="500" height="500" alt="">
                <p><em>Kiwi</em> pie</p>
            </li>
            <li>
                <img src="https://prismic-io.s3.amazonaws.com/api-test/2f4ef8cba993e90635c52dcc4975276a542e45a1_strawberry-pie.png" width="500" height="500" alt="">
                <p><em>Strawberry</em> pie</p>
            </li>
            <li>
                <img src="https://prismic-io.s3.amazonaws.com/api-test/cdc63e583c1a39424b8f16b9bc583ee7ab5de8c4_cherry-pie.png" width="500" height="500" alt="">
                <p><em>Cherry</em> pie</p>
            </li>
        </ul>
    </li>
</ul>
</body>
</html>
